handbook/engineering/extensibility/index.md

```diff
@@ -0,0 +1,66 @@
+# Extensibility team
+
+The extensibility team owns our code host and third-party integrations (including our browser extension) and our [Sourcegraph extensions](https://docs.sourcegraph.com/extensions).
+
+## Contact
+
+- [#extensibility-chat](https://app.slack.com/client/T02FSM7DL/C01LZKLRF0C) channel or @extensibility on Slack
+- On GitHub, mention or assign issues to the [@sourcegraph/extensibility](https://github.com/orgs/sourcegraph/teams/extensibility) team.
+
+## Ownership
+
+This team is responsible for:
+
+1. Bringing the value of Sourcegraph to other developer tools
+1. Bringing the value of other developer tools into Sourcegraph
+
+The team's areas of responsibility include:
+
+- Sourcegraph extensions
+  - Providing a Sourcegraph extension API that enables developers to bring data from their favorite developer tools into their Sourcegraph workflow.
+  - Building useful Sourcegraph extensions on top of the Sourcegraph extension API (for example: Codecov, Datadog, Sentry, Lightstep).
+  - Documentation and tutorials that enable third party developers to create extensions that are actively used.
+  - Building and maintaining the extensions registry and discovery paths to create a community around Sourcegraph extensions.
+- Integrations
+  - Surface code intelligence (and other Sourcegraph data) in code hosts through user installed browser extensions.
+  - Add native support for Sourcegraph in code hosts (for example: GitLab) or popular sites where developers look at code examples (e.g. https://reactjs.org, https://pkg.go.dev/) so users don't have to install our browser extension to get code intelligence.
+
```

Comment:
What about the third category (currently) of IDE --> Sourcegraph extensions? Is that an IDE integration? (If we want the two categories of "in" vs "out" for extensibility?) In which case, should it be a subsection of the above?